src/evm-config.js

```diff
@@ -162,7 +164,7 @@ function sanitizeConfig(name, overwrite = false) {
     };
 
     delete config.origin;
-    changes.push(`replaced superceded 'origin' property with 'remotes' property`);
+    changes.push(`replaced superseded 'origin' property with 'remotes' property`);
```

Comment:
***sigh***

According to my commit logs I've been making this error for literally 20 years now. :older_adult: Apparently it's a blind spot.